src/style.scss

```diff
@@ -1,4 +1,5 @@
 $toolbarHeight: 50px;
+$maxCaptionHeight: 150px;
```

Comment:
I'll take a look at it when I've got a computer to build it on, and that will be about the only thing left that needs checking before merging.